wiki/en/Installation-for-Linux.md

```diff
@@ -67,9 +71,21 @@ Jamulus Clients need [JACK](https://jackaudio.org/) to run, but you need to conf
 
 Restart JACK for the new settings to take effect
 
+### Configure PipeWire
+
+Jamulus Clients need a JACK server included in PipeWire. It is automatically launched when Jamulus is started.
+However, you need to set your audio interface to the "pro-audio" profile and define PipeWire's "rate" and "quantum" parameters beforehand.
+To do so, follow these steps:
+1. open `pavucontrol` (the audio mixer) and choose the "pro-audio" profile in the configuration tab,
+2. In a terminal use the two following commands:
+```bash
+pw-metadata -n settings 0 clock.force-rate 48000
+pw-metadata -n settings 0 clock.force-quantum 128
+```
+
 ### Start Jamulus
 
-With JACK running and configured, launch Jamulus.
+With JACK or PipeWire running and configured, launch Jamulus.
```

Comment:
```suggestion
With JACK or PipeWire configured, launch Jamulus.
```

Reply:
And then a para saying to try with JACK/PipeWire running if running Jamulus didn't start it up automatically.  We'll want bug reports on this because it should just work.